config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+
+  resources :users, only: [:index, :show]do
+    resources :posts, only: [:index, :show]
```

Comment:
- [ ] Please add a `:new` action here to create a route to the form to add a new post
- [ ] Please add the comments and likes routes as well for example:
```ruby
  resources :users do
    resources :posts do
      resources :comments
      resources :likes
    end
  end
```

Reply:
noted, updated with below code:

  resources :comments, only: [:new, :create]
    resources :likes, only: [:create]